packages/tv/tsconfig.json

```diff
@@ -0,0 +1,62 @@
+
```

Comment:
Can we re-use the same config used in `/mobile`? (tweaking it where needed, of course)

Reply:
I really need to learn how tsconfig works better (especially in a bigger project). As long as I don't have a type error I let it be haha but yeah it wasn't needed.